up-l1/zenoh.adoc

```diff
@@ -141,7 +141,7 @@ Note that since authority is case-insensitvie and allows some special characters
 
 [%autowidth]
 |===
-| Speical Characters in authority | Zenoh Key
+| Speical characters in authority | Zenoh Key
```

Comment:
I was actually more concerned with the typo in `Speical` ;-)

Reply:
oh, I missed that typos :laughing: